src/IECoreHoudini/SOP_SceneCacheSource.cpp

```diff
@@ -651,9 +646,14 @@ bool SOP_SceneCacheSource::convertObject( const IECore::Object *object, const st
 				UT_String tag( *it );
 				if ( tag.multiMatch( params.tagFilter ) )
 				{
+					// skip this tag because it's used behind the scenes
+					if ( tag.multiMatch( "ObjectType:*" ) )
```

Comment:
Just a convention, but we always put braces `{}` around `if` bodies, even for one-liners.

Also, I think it might be worth using a static `UT_StringMMPattern` which has been pre-compiled to "ObjectType:*" for this test. They perform much faster than calling `mutliMatch` on a `UT_String`. You probably wouldn't notice in the test cases, but its showed a performance impact in the past (and was the recommendation from SideFx).